src/styles/preview-panel.scss

```diff
@@ -2,6 +2,7 @@
 
 @include meta.load-css("sgds-govtech/css/sgds");
 @include meta.load-css("./isomer-template.scss");
```

Comment:
can we can before/after ss to the PR description followed by test plan (to test this PR + for release to assert what we need to verify looks fine on staging before deploy)?

Reply:
added test plan, will add ss!